pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -648,3 +649,98 @@ func TestBindTwoSbrsWithSingleApplication(t *testing.T) {
 		require.Equal(t, sbrName2, dep.Spec.Template.Spec.Containers[0].EnvFrom[1].SecretRef.LocalObjectReference.Name)
 	})
 }
+
+// TestApplicationByName tests discovery of application by name
+func TestUnBind(t *testing.T) {
```

Comment:
what is scenario we test here, can you reflect with the function name? If the scenario is the same as the new introduced acceptance test, we can then avoid introducing this test  fully.